### ISSUE TITLE
Missing background image leaves empty lockscreen

### ISSUE THREAD

**Issue by DerekSauer** [hyprwm/hyprlock#45]
I'm working on a utility that screenshots my monitors, messes up the images, and uses those for hyprlock's background images. I'm a pretty awful coder so frequently the images aren't where they should be when I test it.

If hyprlock cannot load the background image(s) specified in the config it draws nothing for the background leaving your workspaces visible for the world to see with the widgets drawn on top. 

I think it should probably fallback to the background color specified in the config or the default background color if even that is missing. If a user really wants a transparent lockscreen they can specify a color with a lower alpha value. 

I've written a hack around the problem and will submit a PR, this issue is just for reference.

**Comment by sentakuhm** [comment]
as hypr [wiki](https://wiki.hyprland.org/Hypr-Ecosystem/hyprlock/#background), If path is empty or missing, will use color. Otherwise, the image will be used.
so just change color to solid color.

**Comment by DerekSauer** [comment]
Yeah if the `path` field in the config is empty or missing it does fallback to color as indicated. 

If the image specified by `path` does not exist on disk, it would not render a background at all.

**Comment by sentakuhm** [comment]
> If the image specified by `path` does not exist on disk, it would not render a background at all.

yup you're right.

**Comment by alba4k** [comment]
So I guess you'd like a black background or smth to be rendered? I agree tbh

**Comment by PaideiaDilemma** [comment]
As per #158 if you specify a color in background and the image could not be opened/rendered, then it will fall back to the color.

So I would consider this as fixed.

**Comment by PaideiaDilemma** [comment]
Do we consider this as fixed, or is the expected resolution to always render something, even if no background widget is present?
